docs/HowTo/Monitor/OpenTelemetry-Collector.md

```diff
@@ -0,0 +1,154 @@
+---
+description: Collect Besu information with the OpenTelemetry Collector
+---
+
+
+# Use metrics to monitor node performance
+
+To enable the [Prometheus](https://prometheus.io/) monitoring and tracing service to access
+Hyperledger Besu, use the
+[`--metrics-enabled`](../../Reference/CLI/CLI-Syntax.md#metrics-enabled)
+and [`--metrics-protocol=opentelemetry`](../../Reference/CLI/CLI-Syntax.md#metrics-protocol) options. Use
+[Splunk APM](https://www.splunk.com/en_us/software/splunk-apm.html) or [Splunk](https://splunk.com) to visualize the
```

Comment:
Use definition plugin markup to define `APM`, see https://wiki.hyperledger.org/display/BESU/MkDocs+And+Markdown+Guide#MkDocsAndMarkdownGuide-Abbreviations